build.gradle

```diff
@@ -1,21 +1,24 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 buildscript {
+    apply from: 'versions.gradle'
+    addRepos(repositories)
     repositories {
         jcenter()
         google()
     }
     dependencies {
-        classpath Libs.android_gradle_plugin
-        classpath Libs.dicedmelon_jacoco_plugin
-        classpath Libs.google_oss_licenses_plugin
-        classpath Libs.kotlin_gradle_plugin
+        classpath deps.android_gradle_plugin
+        classpath deps.dicedmelon_jacoco_plugin
+        classpath deps.gms_plugin
+        classpath deps.kotlin.plugin
         // NOTE: Do not place your application dependencies here; they belong
         // in the individual module build.gradle files
     }
 }
 
 allprojects {
+    addRepos(repositories)
```

Comment:
Following `repositories {...}` is replaced by `addRepos(repositories)` so should be no longer needed.